### PR TITLE
Allow configuring log level

### PR DESCRIPTION
**Pull request by pboos**
1. Allow configuring log level
2. Do not report IGNORE level violations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,10 @@ openapi.validation.specification-file-path=/tmp/openapi-spec/openapi.json
 # If it is within src/main/resources/folder/my-spec.json use
 openapi.validation.specification-file-path=folder/my-spec.json
 
+# Custom log level for violations
+# Default: info
+openapi.validation.violation-log-level=error
```
Comment (eleonoraborghi) [inline README.md:93]: should we add an example for other possible values? (warning only is missing I suppose)

**Review by eleonoraborghi — APPROVED** [review]
🥇

File: README.md (hunk 1 above)
Comment (pboos) [reply README.md:93]: There is code completion in the right editors. For me in intellij it tells me which ones I can use :)

Comment (pboos) [reply README.md:93]: but can add it nevertheless :)

**Pull request merged**